packages/cli/e2e-test/cli-e2e-test.js

```diff
@@ -16,6 +16,7 @@
 
 const os = require('os');
 const fs = require('fs-extra');
+const fetch = require('node-fetch');
```

Comment:
I saw whatwg-fetch in the repo, should we use that @benjaminr ?

Reply:
Apologies @benjaminr , I meant @benjdlambert 🙃

Reply:
yeah I was moving more towards using that as it's a one package that can polyfill frontend and backend stuff.

also it plays nicely with `msw` too. but it needs to be version v2 as v3 causes issues as the `react-lazylog` depends on a different version which causes issues when compiling the backend because of package hoisting,

Reply:
Looked into it and it seems like it's actually *not* both backend and frontend at all :)
And not planning to :-P
https://github.com/github/fetch/issues/184